sdk/tests/conformance/extensions/webgl-webcodecs-video-frame.html

```diff
@@ -0,0 +1,224 @@
+<!--
+Copyright (c) 2019 The Khronos Group Inc.
+Use of this source code is governed by an MIT-style license that can be
+found in the LICENSE.txt file.
+-->
+
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="../../resources/js-test-style.css" />
+  <script src="../../js/js-test-pre.js"></script>
+  <script src="../../js/webgl-test-utils.js"></script>
+  <script src="../../js/tests/out-of-bounds-test.js"></script>
+  <script src="../../../../extensions/proposals/WEBGL_webcodecs_video_frame/webgl_webcodecs_video_frame.js"></script>
+  <style>
+    canvas {
+      padding: 10px;
+      background: gold;
+    }
+
+    button {
+      background-color: #555555;
+      border: none;
+      color: white;
+      padding: 15px 32px;
+      width: 150px;
+      text-align: center;
+      display: block;
+      font-size: 16px;
+    }
+  </style>
+</head>
+
+<body>
+  <canvas id="src" width="640" height="480"></canvas>
+  <canvas id="dst" width="640" height="480"></canvas>
+  <p id="info"></p>
+  <div id="description"></div>
+  <div id="console"></div>
+  <script>
+    "use strict";
+    description("Test of importing Videoframe from Webcodecs to Webgl");
+
+    const kIsRunningTest = true;
+    const kMaxFrame = 10;
+    const kTestPixel = [255, 128, 0, 255];
+    // Sum of pixel difference of R/G/B channel. Use to decide whether a 
+    // pixel is matched with another. 
+    const codec_string = "vp09.00.51.08.00";
+
+    let cnv = document.getElementById("src");
+    let src_width = cnv.width;
+    let src_height = cnv.height;
+    let src_color = "rgba(" + kTestPixel[0].toString() + "," + kTestPixel[1].toString() + ","
+      + kTestPixel[2].toString() + "," + kTestPixel[3].toString() + ")";
+    let frame_counter = 0;
+    let pixelCompareThreshold = 10;
+
+    function getQueryVariable(variable) {
+      var query = window.location.search.substring(1);
+      var vars = query.split("&");
+      for (var i = 0; i < vars.length; i++) {
+        var pair = vars[i].split("=");
+        if (pair[0] == variable) { return pair[1]; }
+      }
+      return false;
+    }
+
+    let th = getQueryVariable('th');

```

Comment:
Can we have a more informative name other than 'th'?

Reply:
revise to threshold